iina/MainWindowController.swift

```diff
@@ -1177,6 +1205,7 @@ fileprivate let touchBarItemBinding: [NSTouchBarItemIdentifier: (String, Int, St
 
 @available(OSX 10.12.2, *)
 extension MainWindowController: NSTouchBarDelegate {
+
   
   override func makeTouchBar() -> NSTouchBar? {
     let touchBar = NSTouchBar()
```

Comment:
Please add a `label.alignment = .center` to the text label (when create it, in `case NSTouchBarItemIdentifier.time:`) to make it looks better.